platform-operator/controllers/verrazzano/component/mysql/mysql.go

```
@@ -720,6 +720,9 @@ func grantXARecoverAdmin(ctx spi.ComponentContext) error {
 	if err != nil {
 		return err
 	}
+	if rootPassword == nil {
+		ctx.Log().Progressf("%s is empty in secret %s", mySQLRootKey, rootSec)
+	}
```

probably should return an error

---

Done